SignalMessaging/attachments/AttachmentApprovalViewController.swift

```diff
@@ -12,7 +12,7 @@ public protocol AttachmentApprovalViewControllerDelegate: class {
 }
 
 @objc
-public class AttachmentApprovalViewController: OWSViewController {
+public class AttachmentApprovalViewController: OWSViewController, CaptioningToolbarDelegate {
```

Comment:
<img width="520" alt="screen shot 2017-12-20 at 12 11 14 am" src="https://user-images.githubusercontent.com/217057/34193988-1da6838c-e51c-11e7-9af7-faf54ce5337c.png">

<img width="520" alt="screen shot 2017-12-20 at 12 11 44 am" src="https://user-images.githubusercontent.com/217057/34193998-2695491a-e51c-11e7-8e5c-0b8f54e2af2b.png">

Reply:
Not bad.  I'd argue for bigger margins (between textarea/button + edges of screen) and spacing (between textarea + button).  And I'd vote that the send button's height should match height of empty/single-line textarea height.

Reply:
You're right, I've increased the margins and matched the button height.

<img width="520" alt="screen shot 2017-12-20 at 4 59 37 pm" src="https://user-images.githubusercontent.com/217057/34232731-48a6db64-e5a7-11e7-924b-a6b77e6452a9.png">